src/Resources/views/CRUD/base_edit_form_macro.html.twig

```diff
@@ -8,20 +8,32 @@
             <div class="{{ form_group.box_class }}">
                 <div class="box-header">
                     <h4 class="box-title">
-                        {{ form_group.label|trans({}, form_group.translation_domain ?: admin.translationDomain) }}
+                        {% if form_group.translation_domain is same as(false) %}
+                            {{ form_group.label }}
+                        {% else %}
+                            {{ form_group.label|trans({}, form_group.translation_domain|default(admin.translationDomain)) }}
+                        {% endif %}
                     </h4>
                 </div>
                 <div class="box-body">
                     <div class="sonata-ba-collapsed-fields">
                         {% if form_group.description %}
-                            <p>{{ form_group.description|trans({}, form_group.translation_domain ?: admin.translationDomain)|raw }}</p>
+                            {% if form_group.translation_domain is same as(false) %}
+                                <p>{{ show_group.description|raw }}</p>
+                            {% else %}
+                                {{ form_group.description|trans({}, form_group.translation_domain|default(admin.translationDomain))|raw }}
```

Comment:
should this be wrapped inside `<p>` as before?

Reply:
yes indeed